meter/config_test.go

```diff
@@ -40,3 +43,55 @@ func TestMeters(t *testing.T) {
 		})
 	}
 }
+
+func TestProxyMeters(t *testing.T) {
```

Comment:
können wir das generell in ein proxy_test auslagern und den config_test unangetastet lassen?

Reply:
und die Acceptables dann auch auslagern?

Reply:
Gerne lassen wo die waren

Reply:
aehm. dann muss ich sie im proxy_test duplizieren, nicht schön

Reply:
Ich meinte in der datei, aber zugreifbar